### PR TITLE
Update to 2.10.3 and abi package

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by isuruf** [comment]
@scopatz, looks like these packages have `-2.0` appended to the name so that they can be co-installable and each downstream package has a specific dependency. (Just like gtk2 and gtk3 which are two different conda packages)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,32 +9,49 @@ package:
...
+      run:
+        - sigcpp-{{ abi_version }}  {{ version }}
+    test:
+      commands:
+        - test -f ${PREFIX}/lib/pkg-config/sigc++-{{ abi_version }}.pc
 
 about:
   home: http://libsigc.sourceforge.net/
-  license: LGPL v3
+  license: LGPL-2.1-or-later
```
Comment (scopatz) [inline recipe/meta.yaml]: This should be `LGPL-3.0-or-later`

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**